### PR TITLE
Added image scaling based on speed

### LINKED ISSUE #178: Scale images based on connection speed
The site currently does not perform well on slower connections. The image loader should be updated to pick an image size based on available bandwidth.

### PR DESCRIPTION
**Pull request by icotting**
closes #178

### CODE REVIEW

File: Phocalstream_Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -37,6 +37,42 @@
...
+                $.get("api/photo/speedtest", function (data) {
+                    var now = new Date().getTime();
+                    var timeToLoad = Math.round((now - start)/10)/100;
+                    var connSpeed = Math.round(1024 / timeToLoad/1000);
+
+                    var res = "high";
+                    if ( connSpeed <= 5 ) {
+                        res = "low";
+                    } else if ( connSpeed <= 7 ) {
+                        res = "low";
```
Comment (zmchristensen) [inline Phocalstream_Web/Views/Home/Index.cshtml]: Should this second block be res = "medium"?

**Comment by icotting** [comment]
@zmchristensen, issue resolved with second commit

**Comment by zmchristensen** [comment]
Looks good :+1:

File: Phocalstream_Web/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -128,7 +164,7 @@
     </div>
     
     <div class="background">
-        <img class="background-img" src="/api/photo/high/@Model.Sites.ElementAt(Model.SiteIndex).CoverPhotoID" alt="@Model.Sites.ElementAt(Model.SiteIndex).SiteName">
+        <img class="background-img" src="/api/photo/auto/@Model.Sites.ElementAt(Model.SiteIndex).CoverPhotoID" alt="@Model.Sites.ElementAt(Model.SiteIndex).SiteName">
```
Comment (zmchristensen) [inline Phocalstream_Web/Views/Home/Index.cshtml]: @icotting, we probably want to set this to be `high` regardless of the connection speed. The full screen background image looks pretty blurry at any other resolution.

**Pull request merged**